components/KeyFinancials.vue

```diff
@@ -29,16 +29,90 @@
       'ftsUrl': String,
     },
 
+    data() {
+      return {
+        ftsPlanId: 639,
```

Comment:
lol what am I even doing

Reply:
![it's cool, it's friyay!](https://media.giphy.com/media/xUPN39xtlB15k469TW/giphy.gif)